test/integration/stamp_editor_spec.mjs

```diff
@@ -1096,5 +1096,28 @@ describe("Stamp Editor", () => {
         await page.waitForSelector("#newAltTextDialog", { visible: false });
       }
     });
+
+    it("must check the new alt text flow (part 3)", async () => {
+      // Run sequentially to avoid clipboard issues.
+      for (const [, page] of pages) {
+        await page.evaluate(() => {
+          window.PDFViewerApplication.mlManager.isReadyFlag = false;
```

Comment:
Why not just set `enableAltTextModelDownload` directly here, and avoid changes in the `web/` code just for this one test?
          window.PDFViewerApplication.mlManager.enableAltTextModelDownload = false;
```